lighthouse-core/report/html/report-template.html

```diff
@@ -36,7 +36,7 @@
   </script>
   <script>window.__LIGHTHOUSE_JSON__ = %%LIGHTHOUSE_JSON%%;</script>
   <script>
-    window.addEventListener('DOMContentLoaded', _ => {
+    function __initLighthouseReport__(){
```

Comment:
```suggestion
    function __initLighthouseReport__() {
```
were you raised in a barn